TimeFormat.java

```diff
@@ -1,17 +1,29 @@
 // Represents the hh:mm time format using an AM/PM format. 
 public class TimeFormat {
 	public static void main(String[] args) {
-		// In Java, the command-line arguments args[0], args[1], ... are represented
-		// each by a string. In this program, the single "hh:mm" input is represented
-		// by the single command-line string argument args[0]. 
-		//   
-		// The following statement handles the hours part of the input.
-		// It concatenates the empty string "" with the leftmost hour-digit. 
-		// It then concatenates the resulting string with the rightmost hour-digit,
-		// and then uses parseInt to cast the resulting string as an int.
 		int hours = Integer.parseInt("" + args[0].charAt(0) + args[0].charAt(1));
-		// Does the same with the minutes part of the input.
 		int minutes = Integer.parseInt("" + args[0].charAt(3) + args[0].charAt(4));
-        // Replace this comment with the rest of your code
+		String pmOrAmStr="AM";
+		if (hours<10) {

```

Comment:
Why? do this if.. it doesn't do anything.. 
`If (hours < 10)`
Since `hours` is non-negative number (by definition). 
That means that the cases entering this place are `0,1,2,...,9 `
each of those numbers (lets name them `N` for this discussion)
The statement `N % 10 == N` always holds... Even in general for every `n, m` such that `0 <= n < m` the statement `n % m == n` always holds.
Back to your code, since if condition (hours < 10) is not held the value is kept the same.. and as described above if the condition is held the value is still kept the same.. We can conclude that condition itself is redundant. :)